backend/app/services/diagnosis_agent.py

```
@@ -485,18 +483,12 @@ def norm_t(v):
 
 async def generate_diagnosis(state: DiagnosisState) -> dict:
     from app.core.config import settings
-    api_key, model_name = settings.OPENROUTER_API_KEY, settings.OPENROUTER_PEST_RAG_MODEL
+    api_key, model_name = settings.LITELLM_API_KEY, settings.LITELLM_MODEL
     if not api_key or api_key == "dummy": return {"analysis_result": {"result_text": "API 키 오류"}}
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)